src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -142,13 +142,49 @@ object ScalafixPlugin extends AutoPlugin {
       Invisible
     )
 
+  private val adaptSbtResolvers: Def.Initialize[Task[Seq[Repository]]] =
```

Comment:
Still trying to see how we can avoid making that a task to simplify the implementation and keep completions... Could we [copy the default implementation of `credentials`](https://github.com/search?q=repo%3Asbt%2Fsbt%20sbtCredentialsEnv&type=code) and issue a warning at task execution time if we detect that the user redefined it (in which case the user would have to fallback to `scalafixResolvers` to pass credentials manually).

Reply:
For only god knows reason, for my case, we actually hardcode credentials in the sbt file, so if we go this way, it probably wouldn't work for me :(

But I can understand your concern, this would make implement lots easier, so If there's some dealbreaker, like cache issue you pointed out before, or we can't reach the preferred release date, I think it's fine to release it with only Credential file support.